.circleci/config.yml

```diff
@@ -15,6 +15,11 @@ jobs:
           working_directory: web
           command: npm ci
 
+      - run:
+          name: genproto
+          working_directory: web
+          command: npm run genproto
```

Comment:
I'm wondering if we could have one for GitHub Actions workflow as well?

Reply:
Good point. Done!